config/config_docker.json

```diff
@@ -20,7 +20,9 @@
         "InviteSalt": "gxHVDcKUyP2y1eiyW8S8na1UYQAfq6J6",
         "PublicLinkSalt": "TO3pTyXIZzwHiwyZgGql7lM7DG3zeId4",
         "ResetSalt": "IPxFzSfnDFsNsRafZxz8NaYqFKhf9y2t",
-        "AnalyticsUrl": ""
+        "AnalyticsUrl": "",
+        "UseLocalStorage": true,
+        "StorageDirectory": "/mattermost/"
```

Comment:
maybe /mattermost/data to sit along side bin and config folders?  0/5

now that I think about it I'm not sure what dir docker installers our app in.